layout/netstandard20/project.json

```diff
@@ -2,7 +2,7 @@
   "frameworks": {
     "netstandard1.7": {
       "dependencies": {
-        "NETStandard.Library2": "2.0.0-beta-24801-0"
+        "NetStandard.Library2": "2.0.0-beta-24801-0"
```

Comment:
I'm not sure why do you think this is a typo, if you check [myget feed](https://dotnet.myget.org/feed/dotnet-core/package/nuget/NETStandard.Library2) you'll see how `NETStandard.Library2` is cased correctly